### DISCUSSION TITLE
How can i just update the location of the mesh

### DISCUSSION THREAD

**Discussion by seid111** [JoseConseco/GoB#376] [Q&A]
![image](https://user-images.githubusercontent.com/117804852/200758185-f7b05d7f-8c0a-49f9-abf7-561db589f6cf.png)
![image](https://user-images.githubusercontent.com/117804852/200758677-5a2c0480-be4b-4b97-8c3f-6d268b855f9f.png)

so i had a little diff of the location between zbrush and blender, i dont want them replace my blender mesh, just replace the location of the mesh (bc in zbrush mesh seem not good), what should i do now? tks!

**Comment by kromar** [comment]
Hi, thanks for reporting the issue. I'm not sure i understand the issue from the pictures you posted. Can you explain what you are doing and what exactly the problem is? Also a video or a file where the issue can be reproduced will help.

**Reply by seid111** [reply]
so after i export 1 cube into zbrush(call it A)
in zbrush, i do somestuff like add some surplace sculpt (i call it B)
because i dont want the low poly (A) in blender get some tranforms in surplace like zbrush (bc when i export back, it replace my old low poly A to B) i would like to read the location off the mesh, not the deform of that, sr for my bad explained
![image](https://user-images.githubusercontent.com/117804852/201261441-9c0f7849-9c5a-4588-8710-5989e76060f7.png)
![image](https://user-images.githubusercontent.com/117804852/201261651-e1755427-efc7-4125-839f-8b30bb8d369e.png)

**Reply by seid111** [reply]
btw it not a issue, i just ask some question about my curious, if i post in wrong topic, pls tell me, tks!

**Reply by kromar** [reply]
im not sure i understand yet, so you want your high-poly (B) in a different location than the low-poly (A) but send it to zbrush so both are in the same location in zbrush but in a different location in blender?

**Reply by seid111** [reply]
that not my point, hm like i begin my work with low poly in Blender (i wanna take that to bake), so i export to Zbrush, sculpting v.v, after i done that, some object i wanna change the location, if i change the location in blender and export to zbrush again, they will mess up  the details ( some time the replace detail not play good) so what i want is the object in Zbrush (what i change the location) => export into Blender will just update the location of the mesh in blender, not a whole mesh, so i can ez bake withought worry about them. of course that can make with handly but if i have a lot, that not easy.

**Reply by kromar** [reply]
aha i think i get it now. So you want to change the location in blender without changing it in zbrush and the object position does not change when you send your mesh from zbrush to blender. 
I will need to look into this option but i think it should be possible to do. I let you know when i know more.

**Reply by seid111** [reply]
tysm!!
